packages/charts/src/vaadin-chart-mixin.js

```diff
@@ -1211,6 +1211,12 @@ export const ChartMixin = (superClass) =>
                   effectiveCss += style.textContent;
                 });
 
+                if (self.shadowRoot.adoptedStyleSheets) {
```

Comment:
Might be useful to leave a small comment on which case is handled by this and the block above.

The above could maybe be removed together with removing Polymer support, though I'm not sure if we need to keep supporting injecting stylesheets directly for some other reason.

Reply:
Added comments about Polymer and Lit based versions. The block above can be indeed removed.
There's also one `<style>` tag added by Highcharts itself inside of `<svg>` but it seems not relevant:

```html
<style>
.highcharts-3d-top{filter: url(#highcharts-brighter)}
.highcharts-3d-side{filter: url(#highcharts-darker)}
</style>
```